piqueserver/scripts/savemap.py

```diff
@@ -18,32 +19,52 @@
 from twisted.logger  import Logger
 from pyspades.vxl import VXLData
 from piqueserver.config import config
+from piqueserver.commands import command
 
+savemap_config = config.section('savemap')
 log = Logger()
 
-savemap_config = config.section('savemap')
-LOAD_SAVED_MAP_OPTION = savemap_config.option('load_saved_map', False)
+@command('save', admin_only=True)
+def save(connection):
+    connection.protocol.save_map()
+
+@command('toggle_load_saved_map', admin_only=True)
+def toggle_load_saved_map(connection):
+    protocol = connection.protocol
+    protocol.LOAD_SAVED_MAP_OPTION = not protocol.LOAD_SAVED_MAP_OPTION
+    log.info("LOAD_SAVED_MAP: %s" % protocol.LOAD_SAVED_MAP_OPTION)
+
+@command('toggle_save_at_shutdown', admin_only=True)
+def toggle_save_at_shutdown(connection):
+    protocol = connection.protocol
+    protocol.SAVE_AT_SHUTDOWN_OPTION = not protocol.SAVE_AT_SHUTDOWN_OPTION
+    log.info("SAVE_AT_SHUTDOWN: %s" % protocol.SAVE_AT_SHUTDOWN_OPTION)
 
 
 def get_name(map_info):
     return '%s/%s.saved.vxl' % (os.path.join(config.config_dir, 'maps'),
                                 map_info.rot_info.name)
 
-
 def load_map_from_path(path):
     with open(path, 'rb') as f:
         return VXLData(f)
 
 
 def apply_script(protocol, connection, config):
     class MapSaveProtocol(protocol):
+        SAVE_AT_SHUTDOWN_OPTION = False
```

Comment:
`ALL_CAPS`, in python convention, indicates a constant. This is not one, since it is modified.